cccat/events/event_handlers.py

```diff
@@ -38,6 +41,25 @@ def handle_cat_created(data: dto.JSON) -> None:
     logger.info(f"[{event_id}] Cat {cat_id} has been created")
     # TODO: Handle the async postprocessing of a created Cat here.
 
+    dto_cat_id = dto.CatID(str(cat_id))
+
+    # partial_update = dto.PartialUpdateCat(url="http://placekitten.com/200/300")
```

Comment:
[question] Why is this commented out? The consumer is in charge of deciding what the new URL would be.